shell/platform/fuchsia/flutter/vulkan_surface.cc

```diff
@@ -26,9 +26,21 @@ namespace flutter_runner {
 
 namespace {
 
+// Immutable format is technically limited to R8G8B8A8_SRGB but
+// R8G8B8A8_UNORM works with existing ARM drivers so we allow that
+// until we have a more reliable API for creating external Vulkan
+// images using sysmem. TODO(fxb/52835)
+#if defined(__aarch64__)
 constexpr SkColorType kSkiaColorType = kRGBA_8888_SkColorType;
 constexpr VkFormat kVulkanFormat = VK_FORMAT_R8G8B8A8_UNORM;
 constexpr VkImageCreateFlags kVulkanImageCreateFlags = 0;
+#else
+constexpr SkColorType kSkiaColorType = kBGRA_8888_SkColorType;
+constexpr VkFormat kVulkanFormat = VK_FORMAT_B8G8R8A8_UNORM;
+constexpr VkImageCreateFlags kVulkanImageCreateFlags =
```

Comment:
why is this needed? it was only needed prior to sysmem because we had to match the parameters used by scenic for import

Reply:
Discussed offline. We decided to use multi-format sysmem extensions so that drivers could choose between BGRA and RGBA formats.